src/App.tsx

```diff
@@ -1,12 +1,85 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './components/GoodsList/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goodsList, setGoodsList] = useState([{
+    id: 0,
+    name: '',
+    color: '',
+  }]);
```

Comment:
add Good type to use it here

```suggestion
  const [goods, setGoods] = useState<Good[]>([]);
```

Reply:
Done earlier. Maybe haven't commited